plugin/trino-exchange/pom.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
entry in top level pom.xml which references trino-exchange is missing

Reply:
Yeah, this PR only includes the plugin code. Reason is this PR is against the branch `implement-task-level-retries` and Andrii will continuously update it, which will introduce conflicts. Once Andrii's task-level retry PR merges I will push full code.